tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+    "target": "es2017",
+    "module": "es2015",
+    "lib": ["es2017", "esnext.asynciterable", "dom"],
```

Comment:
We shouldn't need `esnext.asynciterable`. It looks like "lib" takes a default value of omitted so try leaving it out?